src/com/shop/Shop.java

```diff
@@ -65,15 +65,17 @@ private Connection connect() {
 	}
 
 	private void displayMenu() {
-		System.out.println("\n===== MAIN MENU =====");
-		System.out.println("P - List all products.");
-		System.out.println("O – Order a product.");
-		System.out.println("R – Return a product.");
-		System.out.println("S - Search a product by name.");
-		System.out.println("E - List all orders.");
-		System.out.println("C - Current budget.");
-		System.out.println("X - Exit.");
-		System.out.print("\nEnter Menu Option: ");
+		System.out.println("""
+			===== MAIN MENU =====.
```

Comment:
Java version 13 introduced text blocks, reduces the number of println statements and auto inserts \n